apps/demo-app/package.json

```diff
@@ -4,8 +4,8 @@
   "version": "1.0.0-beta.63",
   "type": "module",
   "scripts": {
-    "dev": "turbo run vite --filter demo-app",
-    "vite": "vite",
+    "dev": "turbo run turbo:dev --filter demo-app",
+    "turbo:dev": "vite",
```

Comment:
I would prefer a non-turbo specific command name, e.g.:
- "dev:run"
- "build:watch"
- "watch"

Reply:
I wanted to indicate that the `turbo:dev` script should not be run manually. It is only needed for turborepo.
Would you still rename it?

Reply:
I would prefer it, but it's not a requirement for me